src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -256,9 +256,9 @@ void GraphingCalculator::OnDataRequested(DataTransferManager ^ sender, DataReque
         {
             auto localizedSeperator = LocalizationSettings::GetInstance().GetListSeparator() + L" ";
 
-            rawHtml << L"<span style=\"color: rgb(68, 114, 196); font-style: bold; font-size: 13pt;\">";
+            rawHtml << L"<br><span style=\"color: rgb(68, 114, 196); font-style: bold; font-size: 13pt;\">";
             rawHtml << resourceLoader->GetString(L"VariablesShareHeader")->Data();
-            rawHtml << L"</span><br><span>";
+            rawHtml << L"</span><br><div style=\"margin: 4pt 0pt 0pt 0pt;\"><span>";
```

Comment:
><div style=\"margin: 4pt 0pt 0pt 0pt;\"> [](start = 36, length = 40)

does this do anything beyond putting a margin around the span?  We could just put this style tag in the span itself right?

Reply:
That may work normally, but the mail app seems to ignore a lot of css styling unless it is on specific elements for some reason, so I had to hack whatever I could so that the mail app also respects the margins and other styling.

Reply:
Removed the unnecessary span as discussed!